### PR TITLE
Add controller test for Full-Health-Assessment endpoint

### PR DESCRIPTION
**Pull request by jbregstone**
# Full-Health-Assessment testing 

### What was the problem?
There were no tests for the full-process endpoint in the controller

### How does this fix it?
Adds positive and negative test cases to the controller tests

### Jira Tickets

- [MCP-1806](https://amida.atlassian.net/browse/MCP-1806)

## How to test this PR

- Step 1
- Run tests in controller and see that they pass
- Step 2 
- Manually test the Full-Health-Assessment endpoint on Swagger

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
- [ ] If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) and
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
- [ ] If changing software behavior, post a link to the PR in Slack channel #benefits-virtual-regional-office

### CODE REVIEW

**Review by dimitri-amida — APPROVED** [review]
lgtm

File: controller/src/main/java/gov/va/vro/controller/VroController.java [hunk 1]
```diff
@@ -123,17 +123,26 @@ public ResponseEntity<Object> fetchPdf(String claimSubmissionId)
...
           objectMapper.readValue(responseAsString, FullHealthDataAssessmentResponse.class);
+      if (response.getEvidence() == null) {
+        throw new ClaimProcessingException(
+            claim.getClaimSubmissionId(), HttpStatus.NOT_FOUND, "No evidence found.");
+      }
       log.info("Returning health assessment for: {}", claim.getVeteranIcn());
       response.setVeteranIcn(claim.getVeteranIcn());
       response.setDiagnosticCode(claim.getDiagnosticCode());
       return new ResponseEntity<>(response, HttpStatus.CREATED);
+    } catch (ClaimProcessingException cpe) {
```
Comment (wlin2019) [inline controller/src/main/java/gov/va/vro/controller/VroController.java:144]: The rethrow ClaimProcessingException should not be needed here, should it?

**Review by wlin2019 — APPROVED** [review]
LGTM

File: controller/src/main/java/gov/va/vro/controller/VroController.java (hunk 1 above)
Comment (jbregstone) [reply controller/src/main/java/gov/va/vro/controller/VroController.java:144]: When I remove it I get `500 INTERNAL_SERVER_ERROR` for the missing evidence test

Comment (jbregstone) [reply controller/src/main/java/gov/va/vro/controller/VroController.java:144]: Instead of `NOT_FOUND`

**Pull request merged**